langchain/callbacks/aim_callback.py

```diff
@@ -268,9 +268,13 @@ def on_chain_start(
 
         inputs_res = deepcopy(inputs)
 
-        self._run.track(
-            aim.Text(inputs_res["input"]), name="on_chain_start", context=resp
-        )
+        for inputs_key, inputs_value in inputs_res.items():
+            if isinstance(inputs_value, str):
+                self._run.track(
+                    aim.Text(inputs_value),
+                    name=f"on_chain_start_{inputs_key}",
```

Comment:
@agola11 will anything downstream break if this isn't a fn name?

Reply:
not sure I understand